vulnerabilities/tests/test_cpe_reference.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) nexB Inc. and others. All rights reserved.
```

Comment:
Could it be instead something to add to the to the test_models.py?

Reply:
test_models is not active for now, so making a different test for now, once its active I will add this there